src/services/azblob/backend.rs

```diff
@@ -549,20 +510,18 @@ impl Backend {
 
         let req = isahc::Request::head(&url);
 
-        let mut req = req.body(isahc::AsyncBody::empty()).map_err(|e| {
-            error!("object {path} get_blob_properties: {url} {e:?}");
-            new_request_build_error("stat", path, e)
-        })?;
+        let mut req = req
+            .body(isahc::AsyncBody::empty())
```

Comment:
```suggestion
            .body(AsyncBody::empty())
```
We had `isahc::AsyncBody` imported before, at line 27.

Reply:
Can you raise a PR to address those issues?